docs/adr/0-template_adr.md

```diff
@@ -0,0 +1,23 @@
+# ADR template
+
+Template based on [Documenting architecture decisions - Michael Nygard](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions), [Markdown version](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md) from Joel Parker Henderson.
```

Comment:
The blogpost link is not working for me

Reply:
It does open, eventually... takes really long time :( But that is "the reference" on ADRs (and the one used for this particular format/template). We may also take that whole intro part out, or just leave the link to the Markup template I used. What do you think?

Reply:
I usually use archive.org link: https://web.archive.org/web/20200129074448/http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions works better than the original one

Regarding the use of ADRs - I'm in favour - will provide much-needed context :)

Reply:
We should ask Michael to upgrade his setup :p Anyways, I think staying with ADRs seems the way to go. I can replace the link so it is "responds in reasonable time" for now and as we move forward we can evaluate if we keep this intro or not on the template. Are you OK with that?